lib/utils/response.ts

```diff
@@ -192,6 +193,17 @@ export function ResponseErrorValidation(
   };
 }
 
+/**
+ * Returns a response describing a validation error.
+ */
+export function ResponseErrorValidationFromValidation<T>(
```

Comment:
better naming: ResponseErrorValidationPayload ?

Reply:
I was actually thinking of removing the current `ResponseErrorValidation` and name this one as `ResponseErrorValidation` instead, I have to check if we still use the current one once I've replace all payload validations